src/componenets/LinkAdd.tsx

```diff
@@ -2,14 +2,18 @@ import "./LinkAdd.css";
 import link from "../images/link.svg";
 import { useState } from "react";
 
-function LinkAdd({ toggleModal }) {
+interface ToggleModal {
```

Comment:
타입명 자체는 이렇게 지어주는게 맞습니다~
```suggestion
interface LinkAddProps {
```